crates/five8_core/src/lib.rs

```diff
@@ -1,7 +1,8 @@
 #![doc = include_str!("../README.md")]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#[cfg(feature = "std")]
-extern crate std;
+// Support using the crate without the standard library.
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Alternatively, we can keep:
```rust
#[cfg(feature = "std")]
extern crate std;
```
and just add `#![no_std]`.

Reply:
https://users.rust-lang.org/t/no-std-with-feature-flag-two-ways/90139